dGame/dUtilities/SlashCommandHandler.cpp

```diff
@@ -120,11 +142,6 @@ void SlashCommandHandler::SendAnnouncement(const std::string& title, const std::
 		bitStream.Write<char>(character);
 	}
 
-	bitStream.Write<uint32_t>(message.size());
```

Comment:
Why was this removed?

Reply:
it got removed in the merge i did last night, working on it now